src/store/reducers.js

```diff
@@ -9,7 +9,7 @@ import {
 import systemProfileStore from './systemProfileStore';
 import { applyReducerHash } from '@redhat-cloud-services/frontend-components-utilities/ReducerRegistry';
 import { mergeArraysByKey } from '@redhat-cloud-services/frontend-components-utilities/helpers';
-import { notificationsReducer } from '@redhat-cloud-services/frontend-components-notifications/redux';
+import { notificationsReducer } from '@redhat-cloud-services/frontend-components-notifications';
```

Comment:
I'm not sure this is right. I can't find the `notificationsReducer` export in the notifications package anymore.